pages/games/[id]/index.js

```diff
@@ -46,53 +88,61 @@ const StyledSection = styled.section`
   display: flex;
   flex-direction: column;
   align-items: center;
-  background-color: #e6e6fa;
+  background-color: #f5f5f7;
+  margin-bottom: 5rem;
+`;
+
+const FavouriteButtonWrapper = styled.div`
+  position: relative;
+  left: 38%;
```

Comment:
It would be more common to use position absolute + position relative on the parent element it need to be positioned to relatvely. This might work but I assue that the styling will break quite easily.